dev/SplitButton/SplitButton.cpp

```diff
@@ -318,19 +314,24 @@ void SplitButton::OnSplitButtonKeyUp(const winrt::IInspectable& sender, const wi
             args.Handled(true);
         }
     }
-    else if (key == winrt::VirtualKey::Menu)
-    {
-        m_isAltKeyDown = false;
-    }
     else if (key == winrt::VirtualKey::Down)
     {
-        if (IsEnabled() && m_isAltKeyDown)
+        winrt::CoreVirtualKeyStates menuState = winrt::CoreWindow::GetForCurrentThread().GetKeyState(winrt::VirtualKey::Menu);
```

Comment:
We probably need to do something different for islands to use the XamlRoot instead. Can you ask Austin/Jesse?

Reply:
Thanks! Just as a test i'm going to add @Austin-Lamb and see if he gets notifications about this. :)

Reply:
Notifications work!  And yep, this will work since the key state (and GetAsyncKeyState, if you should ever want that) are really about thread state so this is already good for islands.